client/client.go

```diff
@@ -74,6 +77,9 @@ type Client struct {
 	staticPubKey        glow.PublicKey
 	staticPrivKey       glow.PrivateKey
 
+	// Event log.
+	Log *glow.EventLog
```

Comment:
ohh, should probably call this EventLog so that it's more clear that these logs aren't going to disk by default

Reply:
Need to make sure there are safeties around this ensuring that the EventLog can never consume more than 20 MB of RAM

Reply:
Change the naming